### PR TITLE
Moth recall additions

### PR DESCRIPTION
**Pull request by karlbeckettbjss**
merge MOTH recalls tests into master

### CODE REVIEW

File: src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java [hunk 1]
```diff
@@ -1248,6 +1248,20 @@ private void clickAndWaitForPageLoad(WebElement element) {
         }
     }
 
+    /**
+     * Wait javascript events to complete.
+    */
+    protected void waitForJStoEnd() {
```
Comment (andrew-lai-bjss) [inline src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java]: Is this method used anywhere?

File: src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java [hunk 2]
```diff
@@ -1143,12 +1143,12 @@ public void selectOptionInFieldById(String optionText, String id) {
      */
     public void checkCurrentPageTitle(String expected) {
         String actual = webDriver.getTitle();
-        logger.debug("Checking current page title for '{}' contains '{}'", actual, expected);
-        if (!actual.contains(expected)) {
+        logger.debug("Checking current page title for '{}' contains '{}'", actual, expandDataKeys(expected));
+        if (!actual.contains(expandDataKeys(expected))) {
```
Comment (andrew-lai-bjss) [inline src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java]: Do you actually need / use this?

File: src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java (hunk 1 above)
Comment (karlbeckettbjss) [reply src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java]: It is not, I will remove it

File: src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java (hunk 2 above)
Comment (karlbeckettbjss) [reply src/main/java/uk/gov/dvsa/mot/framework/WebDriverWrapper.java]: This was going to be used but in the end I didn't. I will remove this change

**Pull request merged**